### DISCUSSION TITLE
Hey Kurt!

### DISCUSSION THREAD

**Discussion by hexbyt3** [kwsch/PKHeX#3928] [Show and tell]
Hi Kurt!  I was hoping to get your opinion on the new site I made aimed to help users better configure pokemon with legality support, batch command support, and more!

Your opinion means a lot to me and I hope you like it!

https://genpkm.com/

**Comment by hexbyt3** [comment]
Thanks @sora10pls I'm a noob at Github xD I appreciate you!

**Comment by kwsch** [comment]
Looks fine for what it is; careful about filtering the list of moves when there's potentially special event moves.

**Reply by hexbyt3** [reply]
Thanks for the reply, Kurt!  I still have a ton to work on for sure. But it’s definitely a start and a step in the right direction so far!
